pages/landing/enterprise-eks/index.html

```diff
@@ -109,7 +109,7 @@
       - title: Flux
         src: /assets/img/logos-technologies/logo-flux.png
 
-  - heading: "Step 2: Generate Code"
+  - heading: "Step 2: Generate and Customize Code"
```

Comment:
1. Perhaps "Generate code and customize it to your needs"? 
1. As discussed in Notion, let's remove the folder structure tab, and make the `eks-cluster/main.tf` the first and default tab.
1. Consider tweaking the "Step 1" text to be gray and appear above each heading, as on this page: https://gruntwork.io/patcher/use-cases/

Reply:
Done all but (3) in 2ebb6ed1fd17fc8a6d348f69904ce94b96367174. Will work on (3) in a separate PR

Reply:
(3) is now done in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/624